### DISCUSSION TITLE
Selection-Criterion (SDK 1.12)

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#1013] [Q&A] [answered]
In SDK 1.12 the number of selectable selection criteria has increased considerably.

Is there any correspondence between those in SDK 1.10 and those in SDK 1.12?

SELECTION-CRITERION SDK10:
 
ef-stand -> Solvencia económica y financiera
other -> Otros
sui-act -> Aptitud para ejercer la actividad profesional
tp-abil -> Capacidad técnica y profesional
 
SELECTION-CRITERION SDK12:
 
slc-abil-facil-res -> Instalaciones de estudio, técnicas y de investigación
slc-abil-facil-tools -> Herramientas, planta o equipo técnico
slc-abil-mgmt-env -> Medidas de gestión medioambiental
slc-abil-mgmt-qual -> Medidas para garantizar la calidad
slc-abil-mgmt-supply -> Gestión de la cadena de suministro
slc-abil-qual-inst -> Certificados de institutos de control de calidad
slc-abil-qual-smp-w-autent -> Muestras, descripciones o fotografías con certificación de autenticidad para contratos de suministro
slc-abil-qual-smp-wo-autent -> Muestras, descripciones o fotografías sin certificación de autenticidad
slc-abil-ref-services -> Referencias sobre servicios específicos
slc-abil-ref-supply -> Referencias sobre entregas específicas
slc-abil-ref-work -> Referencias sobre obras específicas
slc-abil-staff-qual -> Titulaciones educativas y profesionales relevantes
slc-abil-staff-tech-ctrl -> Técnicos o entidades técnicas para el control de calidad
slc-abil-staff-tech-work -> Técnicos o entidades técnicas para llevar a cabo el trabajo
slc-abil-staff-yrly-avg-mp -> Media anual de personal
slc-abil-staff-yrly-no-mgmt -> Número de personal directivo
slc-abil-subc -> Proporción de subcontratación
slc-sche-env-cert-indep -> Certificados por organismos independientes sobre sistemas o normas de gestión ambiental
slc-sche-qu-cert-indep -> Certificados por organismos independientes sobre normas de aseguramiento de la calidad
slc-sec-inf -> Seguridad de la información
slc-sec-proc -> Seguridad para procesar, almacenar y transmitir información clasificada
slc-sec-supply -> Seguridad del suministro
slc-stand-ins -> Seguro de indemnización por riesgos profesionales
slc-stand-other -> Otros requisitos económicos o financieros
slc-stand-ratio -> Ratio financiero
slc-stand-to-avg -> Volumen de negocios anual medio
slc-stand-to-gen -> Volumen de negocios anual general
slc-stand-to-spec -> Volumen de negocios anual específico
slc-stand-to-spec-avg -> Volumen de negocios anual medio específico
slc-suit-auth-mbrshp -> Autorización o pertenencia a una organización particular necesaria para contratos de servicios
slc-suit-reg-prof -> Inscripción en un registro profesional relevante
slc-suit-reg-trade -> Inscripción en un registro mercantil

Thanks in advance,
Alex

**Comment by YvesJo** [comment, marked as answer]
Hi,
With SDK 1.12 and to align with the amendment and the change to the way reference to source of information is made, the list has changed from a list of criteria categories to a list of criteria.
Most codes start with a prefix that allow to identify the category it belongs to.
Also note that the details of the selection criteria are only required in the notice when the notice has itself been identified as a source of information for the selection criteria.
